examples/README.md

```diff
@@ -19,25 +19,24 @@ use them to deserialize the contents of the message, by using the format and com
 in the headers.
 
 * Schema validation: after the contents of the message has been extracted, it is validated using the 
-schemas provided by the Redpanda Schema registry. To do that, it reads from the headers ... the subject name
+schemas provided by the Redpanda Schema registry. To do that, it reads from the headers the subject name
 and version number of the schema the message was encoded into. With these two fields it will request
 the Redpanda service for the schema published using that subject and version number. The schema downloaded
-is then processed by the Fastavro library to validate the correction of the message contents.
+is then processed by the [fastavro|https://github.com/fastavro/fastavro] library to validate the correction of the message contents.
 
 * Message processing: if the validation is correct, a new instance of the Processor class defined in 
-the PROCESSORS field of the config is created and the library will call the method process() with the
+the PROCESSORS field of the config is created and the library will call the method ```process()``` with the
```

Comment:
You want single backticks if it's inline code.  You only use triple backticks when you are fencing your code in a separate paragraph.  Like you have elsewhere when the syntax highlighting language is specified after the first fence.